packages/core/src/yjs/extensions/schemaMigration/migrationRules/stripDisallowedMarks.test.ts

```
@@ -0,0 +1,187 @@
+
+    const editor = BlockNoteEditor.create();
+    stripDisallowedMarks(fragment, editor.schema.blockSchema);
+
+    const delta = text.toDelta() as {
+      insert: string;
+      attributes?: Record<string, unknown>;
+    }[];
+    const remaining = new Set<string>();
+    for (const op of delta) {
+      if (op.attributes) {
+        Object.keys(op.attributes).forEach((k) => remaining.add(k));
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)